README.rst

```diff
@@ -57,6 +57,37 @@ can find it at `PULL_REQUEST_TEMPLATE.md <.github/PULL_REQUEST_TEMPLATE.md>`_.
 The issue report template should be automatically applied if you are creating an issue on GitHub as well. Otherwise you
 can find it at `ISSUE_TEMPLATE.md <.github/ISSUE_TEMPLATE.md>`_.
 
+Getting Started
+-----------------
+If you want to run dbt-schema-builder locally, clone this repository on your local system.
+You need a python virtual environment to run this, use the following commands to create and activate python 3.6.12 virtual environment
```

Comment:
Not everyone uses pyenv to manage their virtual environments, it might make sense to call out here that this example uses pyenv, or just say "create a Python 3.6 virtual environment using your favorite tool"